bundles/org.openhab.binding.sensorpush/README.md

```diff
@@ -0,0 +1,74 @@
+# SensorPush Binding
+
+[SensorPush](https://www.sensorpush.com/) sells a line of battery-powered wireless sensors that, depending on the model, provide data on temperature, relative humidity, barometric pressure, dew point, and vapor pressure deficit (VPD).
+The sensors communicate using Bluetooth LE.
+While they can be used directly via BLE, when multiple sensors are in use they are typically configured to relay data to the cloud via the G1 WiFi Gateway.
+This binding retrieves sensor data from the SensorPush Gateway Cloud using a published API.
+
+## Supported Things
+
+Supported sensors: HT1, HT.W, and HTP.XW
+
+The binding requires use of the G1 WiFi gateway and a connection to the SensorPush Gateway Cloud.
+Note that the HTP.XW sensor has not yet been tested.
+
+## Discovery
+
+Automatic discovery is supported for the sensors, but not for the cloud gateway.
+It is recommended that you configure the cloudbridge thing manually and let the associated sensors be discovered automatically.
+
+## Thing Configuration
+
+It is recommended that the SensorPush binding be configured through the management UI.
+There is no easy way for the user to determine the sensor IDs in advance, so it is best to simply auto-discover them.
+After configuring the bridge, all active sensors should appear in the discovery inbox.
+
+### cloudbridge thing
+
+The `cloudbridge` thing is responsible for communicating with the SensorPush Gateway Cloud.
+You must supply your user name and password.
+The poll and timeout parameters are optional.
+
+Parameters:
+
+* `user` (required) Your SensorPush cloud service user name (email address)
+* `password` (required) Your SensorPush cloud service password
+* `poll` (2-60, default = 5) Polling interval in minutes
+* `timeout` (1-120, default = 30) Timeout period for API requests in seconds.
+
+**Note:** To activate your API access, you must log in to the [Gateway Cloud Dashboard](https://dashboard.sensorpush.com/) and agree to the terms of service.
+Once you've logged in that initial time, your account will have access.
+
+### sensor thing
+
+The `sensor` thing represents an individual SensorPush sensor.
```

Comment:
Yes, there are currently 3 models, but they are mostly the same, so it's not really worth defining different things for them. From the binding's perspective, the only difference is that one model provides barometric pressure data and the other two do not. The model also does not seem to be auto-discoverable.

Reply:
What a shame the API doesn't expose the model. I'd name the Thing "genericSensor" or so to make it easier adding more Things in the future in case there will be more sensors with an improved API.